xtokens/src/mock/para.rs

```diff
@@ -270,6 +270,53 @@ parameter_types! {
 	pub SelfLocation: MultiLocation = MultiLocation::new(1, X1(Parachain(ParachainInfo::get().into())));
 	pub const BaseXcmWeight: Weight = 100_000_000;
 	pub const MaxAssetsForTransfer: usize = 3;
+	pub SupportedMultiLocations: Vec<MultiLocation> = vec![
+		MultiLocation::new(
+			1,
+			X1(Parachain(1))
+		),
+		MultiLocation::new(
+			1,
+			X1(Parachain(2))
+		),
+		MultiLocation::new(
+			1,
+			X1(Parachain(3))
+		),
+		// There's a test that uses para id = 100
+		MultiLocation::new(
+			1,
+			X1(Parachain(100))
+		)
+	];
+}
+
+pub struct WhiteListingMultiLocations;
+impl Contains<MultiLocation> for WhiteListingMultiLocations {
```

Comment:
we could offer some default implementation, and use Tuple combine with fall-through way. i.e. `AllowParentAccount`, `AllowParachainAccount`, `AllowLocalAccount`.

Reply:
or a simple way like:
```
match_type! {
	pub type ParentOrParachain: impl Contains<MultiLocation> = {
		MultiLocation { parents: 1, interior: X1(AccountId(_)) } |
		MultiLocation { parents: 1, interior: X2(Parachain(_), AccountId(_)) }
	};
}
```

if allow local asset:
```
match_type! {
	pub type ParentOrParachainOrLocal: impl Contains<MultiLocation> = {
                 MultiLocation { parents: 0, interior: X1(AccountId(_)) } |
		MultiLocation { parents: 1, interior: X1(AccountId(_)) } |
		MultiLocation { parents: 1, interior: X2(Parachain(_), AccountId(_)) }
	};
}
```

Reply:
I cannot use `Parachain(_)`, becasue it will match all para id.